catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -410,53 +455,62 @@ public Metacard transform(InputStream input, String id)
             if (templates != null) {
                 metadataText = transformToXml(metadataText);
             }
+
             String metacardContentType = metadata.get(Metadata.CONTENT_TYPE);
-            MetacardType metacardType = getMetacardTypeFromMimeType(metacardContentType);
-            if (metacardType == null) {
-                metacardType = commonTikaMetacardType;
-            }
-            Metacard metacard;
+            MetacardType extendedMetacardType = getExtendedMetacardType(metacardContentType);
+            Metacard metacard = MetacardCreator.createMetacard(metadata,
+                    id,
+                    metadataText,
+                    extendedMetacardType);
+
             if (textContentHandler != null) {
                 String plainText = textContentHandler.toString();
-
```

Comment:
A lot of the removed code below was moved to a new helper method `getMetacardTypeFromMimeType`